data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/RequestMapper.java

```diff
@@ -119,6 +121,16 @@ public void handle(final HttpServerRequest request) {
       return;
     }
 
+    if (request.method() != HttpMethod.POST) {
+      request.response().setStatusCode(METHOD_NOT_ALLOWED.code()).end();
+
+      logger.warn("method not allowed {}",
+        keyValue("method", request.method())
+      );
```

Comment:
spec says:

> Brokers MUST reject all HTTP produce requests with a method other than POST responding with HTTP status code `405 Method Not Supported`.

Perhaps we use that as some better `WARN` text?

Reply:
How about now?